src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -1620,20 +1620,15 @@ paths:
         Retrieve the results of a bulk file load. The results of each bulk load are stored
         in the dataset. They can be queried directly or retrieved with this paginated
         interface.
-      operationId: bulkFileResultsGet
+      operationId: getLoadHistoryForLoadTag
       parameters:
         - $ref: '#/components/parameters/Id'
-        - name: loadtag
+        - name: loadTag
           in: path
           description: a load tag
           required: true
           schema:
             type: string
-        - name: jobId
-          in: query
-          description: The job id associated with the load
-          schema:
-            type: string
         - name: offset
           in: query
```

Comment:
I actually didn't realize this change would repurpose this endpoint. 

It's possible some teams rely on this endpoint name/the jobId being an option.

Reply:
The endpoint was unimplemented previous to this.

Reply:
Ah, gotcha!